tsl/test/isolation/specs/continuous_aggs_concurrent_refresh.spec

```diff
@@ -182,11 +182,6 @@ setup
     SET SESSION lock_timeout = '500ms';
     SET SESSION deadlock_timeout = '500ms';
 }
-step "L1_lock_threshold_table"
-{
-    LOCK _timescaledb_catalog.continuous_aggs_invalidation_threshold
-    IN ACCESS EXCLUSIVE MODE;
-}
 step "L1_unlock_threshold_table"
```

Comment:
If the lock step is removed, then we shouldn't need the unlock either.

Reply:
removing the unlock does alter the test and leads to different test output